Presentation.Web/app/components/it-system/usage/tabs/it-system-usage-tab-main.controller.ts

```diff
@@ -28,6 +34,9 @@
             $scope.isValidUrl = (url: string) => Kitos.Utility.Validation.isValidExternalReference(url);
             const saveUrlWithOrgId = $scope.autoSaveUrl + "?organizationId=" + user.currentOrganizationId;
 
+            $scope.showDateFields = uiState.isBluePrintNodeAvailable(blueprint.children.frontPage.children.dateFields);
```

Comment:
not dataFields --> usagePeriod... datefields is also a bad word when used in danish bit it is what they want... we should however keep our wording closer to the actual meaning of the data